themes/mcf/components/Footer.tsx

```diff
@@ -0,0 +1,62 @@
+import { SiteWidth } from "@components/panels/SiteWidth";
+
+import { ExternalLink } from "@components/ExternalLink";
+import { LinkWithQuery } from "@components/LinkWithQuery";
+import SocialMediaContent from "./FooterComponents/SocialMediaContent";
+
+const Footer = () => {
+  return (
+    <footer className="flex flex-col bg-grey-400">
+      <div className="py-12">
+        <SiteWidth>
+          <div className="flex flex-col md:flex-row gap-4">
+            <div className="footer__section w-full md:w-1/2 lg:w-2/3" data-cy="footer-mcf">
+              <h2 className="font-greycliff text-2xl font-bold leading-[24px] tracking-[-0.01em] text-left text-headingText text-headingText">
```

Comment:
Use new `Heading` component here, also I've not looked into the new font typeface but if the tracking and letter spacing is consistent with whenever we use it we will want to bake it into the heading component / or css rather than define on the fly like this